### PR TITLE
Update top-level Story-Promo, Grid and Media Indicator

### PR DESCRIPTION
**Pull request by sareh**
No ticket - updating dependencies

**Overall change:** 
Use `@bbc/psammead-story-promo@4.0.0-alpha.8`, `@bbc/psammead-grid@1.1.7` and `@bbc/psammead-media-indicator@4.0.2`

**Code changes:**
- Bump packages in the top level `package.json`
- Pass `dir` prop to Grid stories and Media Indicator used in Story Promo

As a result of this update, the stories that include story-promo have been updated:
http://localhost:8180/?path=/story/components-grid--example-with-top-story-and-regular-promos
Screenshot 2020-02-10 at 16 48 30

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.0.77 | [PR#x](https://github.com/bbc/psammead/pull/x) Update packages. |
```
Comment (sareh) [inline CHANGELOG.md]:
```suggestion
| 2.0.77 | [PR#3107](https://github.com/bbc/psammead/pull/3107) Update packages. |
```

**Review by DenisHdz — APPROVED** [review]

**Review by thekp — APPROVED** [review]

**Comment by amywalkerdev** [comment]
Top story layout looks a bit wrong: https://www.chromaticqa.com/snapshot?appId=5d28eb5ee163f6002046d6fb&id=5e42743e719fc800226c2865

**Comment by DenisHdz** [comment]
Looking as expected now https://www.chromaticqa.com/snapshot?appId=5d28eb5ee163f6002046d6fb&id=5e42906f878b650022249572

**Review by thekp — APPROVED** [review]

**Comment by PriyaKR** [comment]
Looks good to me.Can be merged.

**Pull request merged**